component-runtime-manager/src/main/java/org/talend/sdk/component/runtime/manager/service/ProducerFinderImpl.java

```diff
@@ -88,26 +103,26 @@ private static Object findNext(final Input input) {
 
         @Override
         public boolean hasNext() {
-            synchronized (this.input) {
+            synchronized (input) {
```

Comment:
Why remove "this." ? it help to read code.

Reply:
matter of taste also. As there's no ambiguity on the variable vs a member.